### PR TITLE
Adds a Discord button to the interface.

### PR DESCRIPTION
**Pull request by DominikPanic**
:cl: LaKiller8
add: Discord button added.
/:cl:

dreamseeker_2019-02-10_16-05-22

### CODE REVIEW

File: interface/interface.dm [hunk 1]
```diff
@@ -157,5 +173,5 @@ Admin:
 
 	src << hotkey_mode
 	src << other
-	if(holder)
+	if(check_rights(R_ADMIN, FALSE))
 		src << admin
```
Comment (MrStonedOne) [inline interface/interface.dm]: Undocumented unrelated change.

Please remove this change and move it to its own pr.

Comment (DominikPanic) [reply interface/interface.dm]: If it's a simple file I'm touching I usually clean it up, but sure.

File: interface/interface.dm [hunk 2]
```diff
@@ -47,10 +47,26 @@
 		to_chat(src, "<span class='warning'>The donation URL is not set in the server configuration.</span>")
 	return
 
+
+/client/verb/discord()
+	set name = "Discord"
+	set desc = "Like our server? Buy us and get satisfaction for your efforts."
```
Comment (MrStonedOne) [inline interface/interface.dm]: Forgot to change this, don't think it's even visible though.
